### PR TITLE
Update zh_CN file

### PR DESCRIPTION
**Pull request by TartaricAcid**
I want @SihenZhang to make a check.

### CODE REVIEW

File: resources/assets/enderio/lang/zh_CN.lang [hunk 1]
```diff
@@ -8,30 +8,23 @@ enderio.power.maxOutput=最大输出
...
-item.enderio.itemConduitFacade.hardened.tooltip=硬化: 抵抗破坏以及爆炸
-item.enderio.itemConduitFacade.transparent.tooltip=透明:当使用非固体方块喷涂时，隐藏导管
-item.enderio.itemConduitFacade.transparent.hardened.tooltip=透明:当使用非固体方块喷涂时，隐藏导管；硬化: 抵抗破坏以及爆炸
+item.itemConduitFacade.tooltip.detailed.line1=可放入喷涂机喷涂
+item.itemConduitFacade.tooltip.detailed.line2=右击一个导管可放置
+item.itemConduitFacade.tooltip.detailed.line3=当手里拿着一个扳手或者导管时 
+item.itemConduitFacade.tooltip.detailed.line4=导管框架隐藏
+item.itemConduitFacade.hardened.tooltip=Hardened=硬化: 抵抗破坏以及爆炸
+item.itemConduitFacade.transparent.tooltip=Transparent=透明:当使用非固体方块喷涂时，隐藏导管
+item.itemConduitFacade.transparent.hardened.tooltip=Transparent=透明:当使用非固体方块喷涂时，隐藏导管；硬化: 抵抗破坏以及爆炸
```
Comment (HenryLoenwind) [inline resources/assets/enderio/lang/zh_CN.lang]: These "Hardened="/"Transparent=" are part of the text, I don't think you want them there.

**Comment by TartaricAcid** [comment]
Thank you

File: resources/assets/enderio/lang/zh_CN.lang [hunk 2]
```diff
@@ -1330,16 +1364,23 @@ enderio.top.redstone.true=开
...
-enderio.top.tank.header.output=输出储罐：%s
+enderio.top.tank.header.input=输入槽：%s
+enderio.top.tank.header.output=输出槽：%s
 enderio.top.tank.header.storage=%s
+enderio.top.rf.header.maxin=最大输入:
+enderio.top.rf.header.maxout=最大输出:
+enderio.top.rf.header.avg=I/O:
+enderio.top.rf.value=%s RF/t
+enderio.top.paint.header=涂覆:
+enderio.top.owner.header=猪人:
```
Comment (HenryLoenwind) [inline resources/assets/enderio/lang/zh_CN.lang]: Is Google translate bugging out here? It says "pigs"...

**Comment by TartaricAcid** [comment]
It should be `主人(zhǔ rén)`, sound like `猪人(zhū rén)`, sorry for my carelessness.

File: resources/assets/enderio/lang/zh_CN.lang (hunk 2 above)
Comment (3TUSK) [inline resources/assets/enderio/lang/zh_CN.lang]: Never trust machine translator... for minecraft terminology please refer to minecraft wiki

Comment (HenryLoenwind) [inline resources/assets/enderio/lang/zh_CN.lang]: Yeah, for translating I second this. For checking if a translation may be usable, putting it into Google translate is the fastest way for me. I'm not really interested in checking each and every text, I leave that to the native speakers.

**Pull request closed without merging**